src/components/forms/IdenticonField.js

```diff
@@ -68,7 +67,6 @@ export default class IdenticonField extends React.PureComponent {
       index: newIndex,
       history: newHistory,
     });
-    this.props.onChange(next);
```

Comment:
Removing this breaks the identicon picker in the form. You are no longer able to pick identicons.

This should become:
```js
const { onChange } = this.props; // Should go at top of function

if (onChange) {
  onChange(next);
}
```